templates/kselftest/kselftest.jinja2

```diff
@@ -0,0 +1,25 @@
+- test:
+    timeout:
+      minutes: 55
+
+    definitions:
+    - from: inline
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: timesync-off
+          description: Disable systemd time sync services
+        run:
+          steps:
+          - systemctl stop systemd-timesyncd || true
+      name: timesync-off
+      path: inline/timesync-off.yaml
+
+    - repository: https://github.com/Linaro/test-definitions.git
+      from: git
+      revision: master
+      path: automated/linux/kselftest/kselftest.yaml
+      name: kselftest
+      parameters:
+        TESTPROG_URL: {{ kselftests_url }}
+        SKIPFILE: skipfile-lkft.yaml
```

Comment:
Some skips will be platform-dependent, and there are many more platforms being tested in KernelCI than LKFT.  So we'll need to either have a common file for everyone, or `skipfile-kernelci.yaml` added to the repo, or have a separate file (probably in a fork).

Reply:
Also worth mentioning that KernelCI tracks and reports regressions only, not test cases that have never passed.  So we may be OK without maintaining a full list of tests that won't be fixed, only the broken ones that really don't work or cause some bad side-effects and prevent other tests from running.